res/values-b+es+r419/strings.xml

```diff
@@ -1369,7 +1369,7 @@ Comandos:
     <string name="Allow_Recombine">Permitir recombinar</string>
     <string name="Defenders">Defensores</string>
     <string name="Attackers">Atacantes</string>
-    <string name="MEGA_SPLIT">MEGA SPLIT 64x</string>
+    <string name="MEGA_SPLIT">MEGA SEPARARSE 64x</string>
```

Comment:
Just like the "Crazy Split", people are already used to the previous text, please leave them as they were.